### ISSUE TITLE
Typo

### ISSUE THREAD

**Issue by maufl** [afterlogic/webmail-lite#20]
I found a typo that crashed my installation of webmail lite *without* error messages.
In libraries/afterlogic/common/managers/social/storages/db/storage.php line 36 it says CApiSocailCommandCreatorPostgreSQL but has to be CApiSocialCommandCreatorPostgreSQL. The a an i are swapped. Thus the autoloaders failed to load the class and failed silently!

**Comment by afterlogic** [comment]
Thank you! Have reported it to the developers.

**Comment by afterlogic** [comment]
...In fact, that has fixed a known problem with PostgreSQL support, now it actually works! \o/
Thanks again! :-)

**Comment by maufl** [comment]
It was really difficult to debug. I had to use the register_tick_function method to trace where the script exited. I'm not sure if I made a mistake with the error reporting, but I looked in every log file and turned display_errors on and still did not get an error. So I guess that the class autoloading can actually fail silently without any error reported anywhere. Maybe you want to check that this can not happen anymore so that errors might be easier to find in the future.
